electron/main.ts

```diff
@@ -40,6 +49,54 @@ const openWindow = (url: string) => {
     titleBarStyle: process.platform === "darwin" ? "hidden" : "default",
   });
 
+  const previousTrack = {
+    tooltip: "Previous",
```

Comment:
Maybe we should translate this but it's probably difficult in this file.
Can you add a //ToDo: figure out how to translate the tooltip